platform/docs/src/main/resources/IntContents.adoc

```diff
@@ -88,70 +86,71 @@ Configuration can be performed using the processes described in the Configuring
 |Required
 
 |Enable System Usage Message
-|systemUsageEnabled
+|`systemUsageEnabled`
 |Boolean
 |Turns on a system usage message, which is shown when the Search Application is opened
 |
 |yes
 
 |System Usage Message Title
-|systemUsageTitle
+|`systemUsageTitle`
 |String
 |A title for the system usage Message when the application is opened
 |
 |yes
 
 |System Usage Message
-|systemUsageMessage
+|`systemUsageMessage`
 |String
 |A system usage message to be displayed to the user each time the user opens the application
 |
 |yes
 
 |Show System Usage Message once per session
-|systemUsageOncePerSession
+|`systemUsageOncePerSession`
 |Boolean
 |With this selected, the system usage message will be shown once for each browser session. Uncheck this to have the usage message appear every time the search window is opened or refreshed.
 |true
 |yes
 
 |Header
-|header
+|`header`
 |String
 |Specifies the header text to be rendered on all pages.
 |
 |yes
 
 |Footer
-|footer
+|`footer`
 |String
 |Specifies the footer text to be rendered on all pages.
 |
 |yes
 
 |Text Color
-|color
+|`color`
 |String
-|Specifies the Text Color of the Header and Footer.  Use html css colors or #rrggbb.
+|Specifies the Text Color of the Header and Footer. Use html css colors or `#rrggbb`.
 |
 |yes
 
 |Background Color
-|background
+|`background`
 |String
-|Specifies the Background Color of the Header and Footer.  Use html css colors or #rrggbb.
+|Specifies the Background Color of the Header and Footer. Use html css colors or `#rrggbb`.
 |
 |yes
 
 |===
 
-== Landing Page
+=== Landing Page
 
-The {branding} landing page offers a starting point and general information for a {branding} node. It is accessible at `/(index|home|landing(.htm|html))`.
+The {branding} landing page offers a starting point and general information for a {branding} node.
+It is accessible at `/(index|home|landing(.htm|html))`.
 
-=== Configuration
+==== Configuration
 
-Configuration can be performed using the processes described in the Configuring DDF section. The configurable properties for the landing page configuration are accessed from *Platform -> Landing Page* in the Admin UI.
+The configurable properties for the landing page configuration are accessed from *{branding} Platform -> Landing Page* in the Admin UI.
```

Comment:
Yes. I'm probably going to be changing it to a variable in the next week or so too.